plugins/forms/dialogs/textfield.js

```diff
@@ -168,7 +168,7 @@ CKEDITOR.dialog.add( 'textfield', function( editor ) {
 						var myType = this.getValue();
 
 						if ( elementType != myType ) {
-							var replace = CKEDITOR.dom.element.createFromHtml( '<input type="' + myType + '"></input>', editor.document );
```

Comment:
Can `myType` escape the input tag? Should we htmlEncode it?

Reply:
Encoded since it should never include special characters.